app/components/preprint-title-editor.js

```diff
@@ -8,7 +8,7 @@ const TitleValidation = buildValidations({
             validator('presence', true),
             validator('length', {
                 // minimum length for title?
-                max: 200,
+                max: 512,
```

Comment:
Backend spec had 200 char title increased to 512, so this updates frontend to match.